assets/js/socket.js

```diff
@@ -0,0 +1,62 @@
+// NOTE: The contents of this file will only be executed if
```

Comment:
should take this file out as we're unlikely to need sockets, plus if we're using elm we'd use the elm-phoenix channel library